awx_collection/plugins/modules/ad_hoc_command.py

```diff
@@ -95,6 +95,13 @@
 '''
 
 EXAMPLES = '''
+- name: Launch an Ad Hoc Command waiting for it to finish
+  ad_hoc_command:
+    inventory: Demo Inventory
+    credential: Demo Credential 
```

Comment:
```suggestion
    credential: Demo Credential
```

fix trailing white space to pass flake8

Reply:
@TheRealHaoLiu Thank you for review. I fixed it.